### PR TITLE
MTurk Craftassist HIT Instructions & Analytics Improvements

### PR DESCRIPTION
**Pull request by ethan-carlson**
# Description

This PR includes changes to the html, css, and js of the MTurk craftassist interaction HIT that shorten and serialize the instructions as well as create infrastructure for analytics to be gathered through Mephisto.  There are also minor changes that allow things to work with the recent refactor.

## Type of change

Please check the options that are relevant.

- [ ] Bug fix (non-breaking change that fixes an issue)
- [X] Proposes a change (non-breaking change that isn't necessarily a bug)
- [ ] Refactor
- [X] New feature (non-breaking change that adds a new functionality)
- [ ] Breaking change (fix or feature that would break some existing functionality downstream)
- [ ] This is a unit test
- [ ] Documentation only change
- [ ] Datasets Release
- [ ] Models Release

## Type of requested review

- [X] I want a thorough review of the implementation.
- [ ] I want a high level review. 
- [ ] I want a deep design review.

## Before and After

A detailed list of UX changes and before/after screenshots are [here](https://docs.google.com/document/d/1JB6igdT6mMuCfRDvocmwp0Tt5WVkIuvzx1oeFCdpkkE/edit?pli=1#heading=h.sh9bthvmc140)

In addition to the UX changes, this PR also implements data logging to Mephisto.  The utils.js and browserInfo.js scripts collect a list of what important elements were clicked when, any messages that come up from the dashboard iframe, and the browser and OS info of the user.  These data are downloaded to the local Mephisto DB as a JSON string and are available for parsing locally (the script for HIT analytics will be in a future PR).

# Testing

If you have Mephisto installed, you can run droidlet/tools/crowdsourcing/droidlet_static_html_task/static_test_script.py to view the HIT on localhost with no dashboard.  Or if you're set up with the instructions [here](https://github.com/facebookresearch/fairo/tree/main/droidlet/tools/hitl/nsp_retrain) you can run droidlet/tools/hitl/nsp_retrain/main.py with a sandbox Mephisto requester and get the full experience.

# Checklist:

- [X] I have performed manual end-to-end testing of the feature in my environment.
- [X] I have added Docstrings and comments to the code.
- [ ] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [X] New and existing unit tests pass locally with my changes.
- [X] I have added relevant collaborators to review the PR before merge.
- [ ] [Polymetis only] I have verified that all scripts in `tests/scripts` runs on hardware.

### CODE REVIEW

File: droidlet/tools/crowdsourcing/droidlet_static_html_task/conf/example.yaml [hunk 1]
```diff
@@ -7,14 +7,16 @@ mephisto:
...
+    maximum_units_per_worker: 5
+    task_tags: "assistant, game, interact"
     task_name: "droidlet-dashboard"
-  architect:
-    heroku_config_args:
-      AWS_ACCESS_KEY_ID: ${env:AWS_ACCESS_KEY_ID}
-      AWS_SECRET_ACCESS_KEY: ${env:AWS_SECRET_ACCESS_KEY}
-      AWS_DEFAULT_REGION: "us-west-1"
-    heroku_app_name: craftassist
+  # architect:
```
Comment (kavyasrinet) [inline droidlet/tools/crowdsourcing/droidlet_static_html_task/conf/example.yaml:17]: nit: remove comments ?

File: droidlet/tools/crowdsourcing/droidlet_static_html_task/server_files/craftassist_task_preview.html [hunk 2]
```diff
@@ -55,107 +56,192 @@
...
+            <li><b>Resume</b> (an earlier action)</li>
+          </ul>
+        </p>
+      </div>
+    </div>
+    <!-- End Commands Reminder -->
+    <div class="col-xs-10">
+      <iframe style="width:100%; height:800px;" src="https://${subdomain}.craftassist.io/"></iframe>
+    </div>
+  </div>
```
Comment (kavyasrinet) [inline droidlet/tools/crowdsourcing/droidlet_static_html_task/server_files/craftassist_task_preview.html:211]: The instructions look great, thanks for adding these in!

**Review by kavyasrinet — APPROVED** [review]
I love the updated instructions in this PR. Thanks for adding these in.

File: droidlet/tools/crowdsourcing/droidlet_static_html_task/conf/example.yaml (hunk 1 above)
Comment (ethan-carlson) [reply droidlet/tools/crowdsourcing/droidlet_static_html_task/conf/example.yaml]: done, thanks

**Review by snyxan — APPROVED** [review]

**Pull request merged**